ee/tasks/usage_report.py

```diff
@@ -187,7 +188,7 @@ def get_org_usage_report(period: Tuple[datetime, datetime], team_ids: List[str])
             multiple_ids_per_person=count_total_persons_with_multiple_ids(team_id),
             group_types_total=GroupTypeMapping.objects.filter(team_id=team_id).count(),
```

Comment:
What about taking all of these out and run them before the dataclass is created, as we said earlier?